elcid/__init__.py

```diff
@@ -80,6 +80,6 @@ def get_menu_items(cls, user):
         if standard_add_patient_menu_item.for_user(user):
             menu_items.append(standard_add_patient_menu_item)
         else:
-            menu_items = [i for i in menu_items if not i.href == "/#/list/"]
```

Comment:
I know this was done, so that TB viewers would see the patient lists. I'm not sure why it was left excludeing`/`